Client/Frontend/Browser/Onboarding/Welcome/PlaylistOnboardingView.swift

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+
+struct PlaylistOnboardingView: View {
+    var onButtonPressed: (() -> Void)?
+    
+    var body: some View {
+        HStack(alignment: .top, spacing: 14.0) {
+            Image("welcome-view-playlist-brave-logo")
+            
+            VStack(alignment: .leading, spacing: 16.0) {
+                VStack(alignment: .leading, spacing: 4.0) {
+                    Text("Ad block is just the beginning")
+                        .font(.title3.weight(.medium))
+                        .foregroundColor(Color(.bravePrimary))
+                    
+                    Text("With Brave Playlist, you can play videos in the background, picture-in-picture, or even offline. And, of course, ad-free.")
```

Comment:
Localize it

Reply:
Fixed at This is a color we dont haveut I added this color to UX struct and used rgb initializer